Packages/Sources/RootView/Home.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  SwiftUIView.swift
+//
+//
+//  Created by HH on 09/01/2024.
+//
+
+import NodeCryptoCore
+import SwiftUI
+
+struct Home: View {
+  @State private var activeTab: Tab = .search
+  @Namespace var animation
+
+  var body: some View {
+    VStack(spacing: 0) {
+      TabView(selection: $activeTab) {
+        Color.red
+          .tag(Tab.home)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Services")
+          .tag(Tab.search)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Notifications")
+          .tag(Tab.notifications)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Profile")
+          .tag(Tab.profile)
+          .toolbar(.hidden, for: .tabBar)
+      }
+
+      CustomTabBar()
+    }
+  }
+
+  @ViewBuilder
+  func CustomTabBar(tint: Color = .blue, inactiveTint: Color = .blue) -> some View {
+    HStack(spacing: 0) {
+
+      TabItem(
+        tint: tint,
+        inactiveTint: inactiveTint,
+        tab: .home,
+        activeTab: $activeTab,
+        animation: animation
+      )
+
+      TabItem(
+        tint: tint,
+        inactiveTint: inactiveTint,
+        tab: .search,
+        activeTab: $activeTab,
+        animation: animation
+      )
+
+      Button(
+        action: {
+
+        },
+        label: {
+          Image(ImageResource.add)
+            .padding(12)
+            .background(
+              Circle()
+                .foregroundStyle(Color.primary1)
+            )
+        }
+      )
+      .frame(maxWidth: .infinity)
+
+      TabItem(
+        tint: tint,
+        inactiveTint: inactiveTint,
+        tab: .notifications,
+        activeTab: $activeTab,
+        animation: animation
+      )
+
+      TabItem(
+        tint: tint,
+        inactiveTint: inactiveTint,
+        tab: .profile,
+        activeTab: $activeTab,
+        animation: animation
+      )
+
+    }
+    .animation(.easeIn, value: activeTab)
+  }
+}
+
+struct TabItem: View {
+  var tint: Color
```

Comment:
I am probably old fashioned oop fan, but I tend to mark everything not to be used outside of the class as `private` - although it means to make manual initializer. It gives me confidence that no one will do something harmful here from outside.